src/python/pants/option/arg_splitter_test.py

```diff
@@ -56,35 +68,11 @@ def _split(self, args_str, expected_goals, expected_scope_to_flags, expected_pos
                        splitter.help_request.all_scopes))
     self.assertEqual(expected_unknown_scopes, split_args.unknown_scopes)
 
-  def _split_help(self, args_str, expected_goals, expected_scope_to_flags, expected_target_specs,
-                  expected_help_advanced=False, expected_help_all=False):
-    self._split(args_str, expected_goals, expected_scope_to_flags, expected_target_specs,
-                expected_passthru=None, expected_passthru_owner=None,
-                expected_is_help=True,
-                expected_help_advanced=expected_help_advanced,
-                expected_help_all=expected_help_all)
-
-  def _split_version_request(self, args_str):
-    splitter = ArgSplitter(ArgSplitterTest._known_scope_infos)
-    splitter.split_args(shlex.split(args_str))
-    self.assertTrue(isinstance(splitter.help_request, VersionHelp))
-
-  def _split_unknown_goal(self, args_str, unknown_goals):
-    splitter = ArgSplitter(ArgSplitterTest._known_scope_infos)
-    result = splitter.split_args(shlex.split(args_str))
-    self.assertTrue(isinstance(splitter.help_request, UnknownGoalHelp))
-    self.assertSetEqual(set(unknown_goals), set(splitter.help_request.unknown_goals))
-    self.assertEqual(result.unknown_scopes, unknown_goals)
-
-  def _split_no_goal(self, args_str):
-    splitter = ArgSplitter(ArgSplitterTest._known_scope_infos)
-    splitter.split_args(shlex.split(args_str))
-    self.assertTrue(isinstance(splitter.help_request, NoGoalHelp))
```

Comment:
These were all in-lined. They weren't being used by more than one test each, so are better placed closer to the actual usage.